src/writer/direntPool.h

```diff
@@ -26,6 +26,15 @@
 namespace zim
 {
   namespace writer {
+    static void destroyPoolBlock(Dirent* pool, uint16_t count=0xFFFF) {
+      for (auto i = 0U; i < count; i++) {
+        try {
+          pool[i].~Dirent();
+        } catch (...){ /*discard*/ }
+      }
+      delete [] (reinterpret_cast<char*>(pool));
+    }
+
```

Comment:
I meant this to be a static member function. Then it won't be duplicated in each translation unit where `direntPool.h` is included.

Reply:
Done